python/enthic/arbre_compte_de_resultat.py

```diff
@@ -0,0 +1,385 @@
+# -*- coding: utf-8 -*-
+"""
+============================================================
+Tree view of data contained in a french "compte de resultat"
```

Comment:
Change the file name of this module to English, to be consistent with the other module.

Reply:
J'ai renommé en "tree_french_compte_de_resultat.py", mais c'est de la comptabilité française, donc y'a forcément du spécifique français